src/components/Table/GridTable.stories.tsx

```diff
@@ -218,13 +223,18 @@ export function NestedCards() {
       content: <div css={Css.xs.$}>{row.name}</div>,
       value: row.name,
     }),
+    add: () => "Add",
   };
   const spacing = { brPx: 4, pxPx: 8, spacerPx: 8 };
   const nestedStyle: GridStyle = {
     nestedCards: {
-      parent: { bgColor: Palette.Gray500, ...spacing },
-      child: { bgColor: Palette.Gray200, bColor: Palette.Gray600, ...spacing },
-      grandChild: { bgColor: Palette.Green200, bColor: Palette.Green400, ...spacing },
+      topLevelSpacerPx: 8,
```

Comment:
Besides allowing a `kind: add` that doesn't have an entry in the `nestedCards` config (i.e. to tell us its not a card), I also changed how the spacer determines its height.

It used to be:

* If we just visited a row, the spacer was that row's `card.spacerPx`

But that was kinda weird, i.e. if I was in a parent card, and then went "child kind 1, child kind 2, child kind 1", the spacer would depend on which card and not "I'm in the parent card, all spacers between my children is X".

But the reason I'd done that is that we also need a "top-level" spacer, i.e. when there are no cards open (we have just closed the top-level card, and are going to show the nest top-level card), there is no "current open card" and we can't do `openCard.spacerPx`.

Hence adding just a new `topLevelSpacerPx`. It's the spacing between the top-most cards.